packages/dtos/src/info.ts

```diff
@@ -13,7 +13,7 @@ export class ActorInfoWithReasonDto {
   @ApiProperty()
   rejectedReason: string
 
-  @ApiProperty()
+  @ApiProperty({ type: ActorInfoDto, isArray: true })
   data: ActorInfoDto
```

Comment:
Should be array type or not an array at all ?
```suggestion
  @ApiProperty({ type: ActorInfoDto, isArray: true })
  data: ActorInfoDto[]
```